### ISSUE TITLE
lastest picture replace the previous picture after uploading.

### ISSUE THREAD

**Issue by blackknb** [froala/wysiwyg-editor#3804]
Hi,

I'm testing the froala editor 3.1.0, after upload a picture by click & selecting, I plan to upload another picture immediately, not after typing some words,

##### Expected behavior.
after upload, there should be two pictures

##### Actual behavior.
the lastest picture replace the previous picture in editor

##### Steps to reproduce the problem.
I've tested in Froala official website, same problem. https://www.froala.com/wysiwyg-editor
However, If I use arrow key move left or right or type some words, after upload the 1st picture, everything is fine. it won't be replaced after upload the 2nd picture.

##### Editor version.
3.1.0

##### OS.
Windows 10

##### Browser.
Chrome 79.0.3945.117

##### Recording.
I hope I've describe it correctly and clearly. Thanks.

**Comment by mik-ro** [comment]
I have the same problem.

**Comment by frankrue** [comment]
Same here.

**Comment by teamboot88** [comment]
same

**Comment by lbelling** [comment]
Hi All, we have a very large customer that is looking to replace their use of Powerpoint with a tool featuring the Froala editor.  However, this outstanding bug is a showstopper.
Adding a new image should never overwrite an existing image.
I ask that you elevate the priority on this and get it fixed asap.
Thank you!

**Comment by tavosansal** [comment]
We are also having this issue. Can get very frustrating for users.

**Comment by joostelders** [comment]
Updating to `3.1.1` resolved this issue for me.

No mentions of it in the changelog though: https://froala.com/wysiwyg-editor/changelog/#3.1.1

**Comment by lbelling** [comment]
we also updated to 3.1.1 and it did not resolve this issue

**Comment by PavelBezpalov** [comment]
Also tried 3.1.1 with no luck. Confirmed.

**Comment by PavelBezpalov** [comment]
To fix issue remove text `&&(function t(){return fe}()||!0===e)` from minified version.
This change simplified check in function from: 
```
    function _exitEdit(force_exit) {
      if ($current_image && (_canExit() || force_exit === true)) {
```

to 
```
    function _exitEdit(force_exit) {
      if ($current_image) {
```

**Comment by liasica** [comment]
Same here +1

**Comment by liasica** [comment]
> To fix issue remove text `&&(function t(){return fe}()||!0===e)` from minified version.
> This change simplified check in function from:
> 
> ```
>     function _exitEdit(force_exit) {
>       if ($current_image && (_canExit() || force_exit === true)) {
> ```
> 
> to
> 
> ```
>     function _exitEdit(force_exit) {
>       if ($current_image) {
> ```

Thank you

**Comment by dejanmartinovic** [comment]
This issue should be fixed in version 3.2.1.

**Comment by ilyaskarim** [comment]
Issue fixed in  3.2.1.
